databases/databases/databases_and_sql.md

```diff
@@ -107,14 +72,14 @@ A close cousin of `SELECT`, for if you only want unique values of a column, is `
 
 If you want to get all the posts created by a given user, you need to tell SQL which columns it should use to zip the tables together with the `ON` clause. Perform the "zipping" with the `JOIN` command.  But wait, if you mash two tables together where the data doesn't perfectly match up (e.g. there are multiple posts for one user), which rows do you actually keep?  There are four different possibilities:
 
-Note: the "left" table is the original table (the one that the `FROM` clause was `ON`), e.g. "users" in examples below.
+_(__note:__ the "left" table is the original table (the one that the `FROM` clause was `ON`), e.g. "users" in examples below.)_
```

Comment:
I haven't seen this before (`_(__note:__`), is this a shortcut to get the info box? Everytime I've used that I've done it with plain HTML

Reply:
So in #24704 and #24705 I added notes with such formatting, I believe I took the idea from here:
https://github.com/TheOdinProject/curriculum/blob/main/foundations/installations/installations.md?plain=1#L120
Note that just after it there's a note with different formatting and it is a lil' broken. I unify how these notes look in #24776 in an attempt to make those notes more uniform and readable by making them start from a newline.

Possibly something to introduce in the Style Guide if you guys figure out which way you want to go.

Reply:
Oh also, no, this is not the info box. The `_` before and after the `(`, `)` make the text italic, while `__note:__` makes it bold.

See `Additional information` of this PR:
3. Additional Information:
Possibly decide whether `_` should be used for italics - I followed the example of previous _(__note:__)_ occurrences but `*` is used in Markdown as well, so might want to clear this up just like with `-` for unordered lists.